cpr/session.cpp

```diff
@@ -365,7 +365,9 @@ void Session::Impl::SetSslOptions(const SslOptions& opts) {
 #endif
     curl_easy_setopt(curl_->handle, CURLOPT_SSL_VERIFYPEER, opts.verify_peer ? ON : OFF);
     curl_easy_setopt(curl_->handle, CURLOPT_SSL_VERIFYHOST, opts.verify_host ? 2L : 0L);
+#if LIBCURL_VERSION_NUM >= 0x074100
```

Comment:
```suggestion
#if LIBCURL_VERSION_NUM >= 0x072900
```
Available since 7.41.0 -> 0x07, 0x29, 0x00
Curl uses hex numbers here ;)

https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYSTATUS.html

https://curl.haxx.se/docs/versions.html

Reply:
ha thanks